lib/active_model_serializers/key_transform.rb

```diff
@@ -66,5 +66,28 @@ def underscore(value)
     def unaltered(value)
       value
     end
+
+    # Transforms string to selected case
+    # Accepts string in any case: 'camel', 'undescore', 'dashed'.
+    #
+    # @example:
+    #   transform('SomeClass', :underscore) => 'some_class'
+    #   transform('some_class', :snake) => 'someClass'
+    #   etc...
+    def transform(string, type)
```

Comment:
removed. I was confused about which `transform` method we were talking.

I need to stop coding when I'm tired...

:embarassed: